cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -53,15 +67,58 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
           resultTable()
         }
       }
+      section("Changing the logged in users password", "administration-security-users-alter-password") {
```

Comment:
I prefer the terms `current user's` or `own`.

Reply:
Although I now see that 'current user' and 'own' are used directly below, so perhaps you wanted three ways of saying this directly after each other so the user is completely clear on what it means?